pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PexExtension.java

```diff
@@ -15,43 +15,42 @@
  */
 package com.linkedin.gradle.python.extension;
 
-import com.linkedin.gradle.python.util.OperatingSystem;
 import org.gradle.api.Project;
 
 import java.io.File;
 
 
-public class PexExtension {
-
-    private File pexCache;
-    private boolean fatPex = OperatingSystem.current().isWindows(); //Defaulting to fat pex's on windows
```

Comment:
Where is this captured now?

Reply:
The `ZipappExtension` class referenced above :)